src/commands/System/help.js

```diff
@@ -27,6 +27,7 @@ module.exports = class extends Command {
 		const helpMessage = [];
 		for (let cat = 0; cat < categories.length; cat++) {
 			helpMessage.push(`**${categories[cat]} Commands**: \`\`\`asciidoc`);
+			helpMessage.push('');
```

Comment:
Instead of calling the push method twice, you can push two values into the same array in a single call.